app/src/main/java/org/p2p/wallet/auth/gateway/repository/GatewayServiceRestoreWalletMapper.kt

```diff
@@ -0,0 +1,88 @@
+package org.p2p.wallet.auth.gateway.repository
+
+import org.near.borshj.Borsh
+import org.p2p.wallet.auth.gateway.api.request.ConfirmRestoreWalletRequest
+import org.p2p.wallet.auth.gateway.api.request.GatewayServiceJsonRpcMethod
+import org.p2p.wallet.auth.gateway.api.request.GatewayServiceRequest
+import org.p2p.wallet.auth.gateway.api.request.OtpMethod
+import org.p2p.wallet.auth.gateway.api.request.RestoreWalletRequest
+import org.p2p.wallet.utils.Base58String
+import org.p2p.wallet.utils.Constants
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+class GatewayServiceRestoreWalletMapper(
+    private val signatureFieldGenerator: GatewayServiceSignatureFieldGenerator
+) {
+    private class RestoreWalletSignatureStruct(
+        val restoreId: String,
+        val appHash: String,
+        val phone: String,
+        val channelType: String
+    ) : Borsh
+
+    private class ConfirmRestoreWalletSignatureStruct(
+        val restoreId: String,
+        val phone: String,
+        val otpConfirmationCode: String
+    ) : Borsh
+
+    fun toRestoreWalletNetwork(
```

Comment:
```suggestion
    fun toNetwork(
```

The mapper class already indicates that this is about Restore Wallet or do you have method conflicts?

Reply:
i need to create requests for two methods from Service, and yeah, i have method conflicts due to that